scm/driver/azure/azure.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 Drone.IO Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package azure implements a azure client.
+package azure
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/drone/go-scm/scm"
+)
+
+// New returns a new azure API client.
+func New(uri, azureOwner, azureProject string) (*scm.Client, error) {
```

Comment:
we could probably rename to `owner` and `project`, where `azure` is inferred from the package name.

Reply:
done